esmvaltool/diag_scripts/cvdp/cvdp_wrapper.py

```diff
@@ -0,0 +1,140 @@
+"""wrapper diagnostic for the NCAR CVDP (p)ackage."""
+import logging
+import os
+import re
+import shutil
+import subprocess
+
+from esmvaltool._task import DiagnosticError
+from esmvaltool.diag_scripts.shared import (group_metadata, run_diagnostic,
+                                            select_metadata)
+
+logger = logging.getLogger(os.path.basename(__file__))
+
+
+def setup_driver(cfg):
+    """Write the driver.ncl file of the cvdp package."""
+    cvdp_root = os.path.join(os.path.dirname(__file__), 'cvdp')
+    if not os.path.isdir(cvdp_root):
+        raise DiagnosticError("CVDP is not available.")
+
+    settings = {
+        'outdir': "{0}/".format(cfg['work_dir']),
+        'obs': 'False',
+        'zp': os.path.join(cvdp_root, "ncl_scripts/"),
+        'run_style': 'serial',
+        'webpage_title': 'CVDP run via ESMValTool'
+    }
+    settings['output_data'] = "True" if _nco_available() else "False"
+
+    def _update_settings(line):
+
+        for key, value in settings.items():
+            pattern = r'\s*{0}\s*=.*\n'.format(key)
+            search_results = re.findall(pattern, line)
+            if search_results == []:
+                continue
+            return re.sub(
+                r'".+?"', '"{0}"'.format(value), search_results[0], count=1)
+
+        return line
+
+    content = []
+    driver = os.path.join(cvdp_root, "driver.ncl")
+
+    with open(driver, 'r') as driver_file:
+        for line in driver_file:
+            content.append(_update_settings(line))
+
+    new_driver = os.path.join(cfg['work_dir'], "driver.ncl")
```

Comment:
I think `cfg['run_dir']` would be more appropriate for this kind of file, the work dir is more for results.

Reply:
You are right! Done